cd/lib/jwt.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+#
+# JWT Encoder Bash Script; adapted from https://www.willhaley.com/blog/generate-jwt-with-bash/
```

Comment:
Any idea if it's ok to use code from a blog post? Is there any license attached to it? maybe we could reach out to the person and ask them?

Reply:
Not sure; the original blog post (from 2016) has no license attached to the code.  

TBH this script is not even necessary for this change as it stands now (was a remnant from an earlier commit); I left it in as it could conceivably be used at some point in the future for authenticating via the GH app (see https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps).  However if there's doubt about whether it's OK to include this, I'd prefer to just remove the script entirely.

Reply:
I'm not super sure with this either. I don't want to block you here, and i'm not again keeping this file here to use it in the future. I just know that to use third party software we need to publish an `ATTRIBUTIONS.md` file containing their respective licenses. e.g https://github.com/aws-controllers-k8s/runtime/blob/main/ATTRIBUTION.md

Reply:
Removed the script, can just figure out the license question later if we end up wanting to use JWT